docs/usermanual-buffers-language-script-and-direction.xml

```diff
@@ -302,7 +302,9 @@
     <para>
       At build time, HarfBuzz looks first for the GLib library. If
       it is found, HarfBuzz will use GLib's Unicode functions by
-      default. If there is no GLib, HarfBuzz will fall back to its own
+      default. If there is no GLib, HarfBuzz will look for the
+      International Components for Unicode (ICU) library. If neither
+      GLib nor ICU is found, HarfBuzz will fall back to its own
```

Comment:
Actually, it's been a while since we prefer our own implementation to that of GLib or ICU.  And as of yesterday, our is always compiled in using the existing build systems.

Reply:
Ah, timing....

Reply:
So: if you enable GLib & ICU config switches, how does that affect the ufuncs selection scheme?

Reply:
Still internal preferred.  Unless turned off.  Don't document how to turn it off.  That's reserved for systems that custom-integrate / build.  Tell them to refer to hb-unicode.cc if needed.  Here's the exact logic:

https://github.com/harfbuzz/harfbuzz/blob/master/src/hb-unicode.cc#L135

Reply:
Over time the glib/icu callbacks have become less relevant for sure.

Reply:
OK. Now just mentioning they exist, and trusting that any interested parties will enjoy exploring it.